src/kvirc/ui/KviWindow.cpp

```diff
@@ -324,15 +324,14 @@ const char * KviWindow::m_typeTable[TypeCount] = {
 	"dccvideo",     // 15
 	"userwindow",   // 16
 	"tool",         // 17
-	"iograph",      // 18
```

Comment:
Not on purpose, c&p monster strikes again.

![cookiemonster](https://cloud.githubusercontent.com/assets/3521959/16945439/fed19626-4d9c-11e6-928b-7802aeba6efe.gif)